### ISSUE TITLE
Lecture 1: Neural Networks and Deep Learning - week1

### ISSUE THREAD

**Issue by KimDahye** [KimDahye/deeplearning_study#1]
- 스터디 일시: 2021.08.18
- 참여자: 고예슬, 김다혜, 서원국, 선승현, 신윤서
- ice breaking
  - 자기소개
  - 스터디 참가 이유

### Learning Objectives
- Discuss the major trends driving the rise of deep learning.
- Explain how deep learning is applied to supervised learning
- List the major categories of models (CNNs, RNNs, etc.), and when they should be applied
- Assess appropriate use cases for deep learning

### 강의자료
- [week1 pdf](https://d3c33hcgiwev3.cloudfront.net/0QkmLH9eRj-JJix_XvY_8g_cd513a39d322432394d1b2353c6cc9a1_C1W1.pdf?Expires=1629417600&Signature=VgyDjpAfbOagPhJrr~o6oHp~Nsikuous3BzIJ1cc4odIeuLCiO6n5zct91kpp66NocD0vGa8XF4spZAJ-SFtMTDvkTPkktJHd3OJ6F9qhVk3bAZH9nYiw~PKFns8S8sEe9Irgerpz4kN0I4L6QoZrDBi7CSAvyEWI69xbH-y7IE_&Key-Pair-Id=APKAJLTNE6QMUY6HBC5A)

### 정리
- 요즘의 AI는 전기가 발명된 것 정도의 혁신을 가져왔다.
- 신경망이란?
  > <img width="600" alt="Screen Shot 2021-08-18 at 8 54 31 PM" src="https://user-images.githubusercontent.com/6873655/129893720-e344b9f8-0efc-4001-b930-f48b73d575d7.png">
  > <img width="600" alt="Screen Shot 2021-08-18 at 8 54 43 PM" src="https://user-images.githubusercontent.com/6873655/129893792-7bdf69fb-a3b3-4008-b6cd-e6475dc10588.png">
  - ReLU
  - 딥러닝이란? 신경망(Neural Network) 트레이닝을 일컫는 말 or 매우 큰 신경망
- Supervised Learning with Neural Networks
  - 딥러닝이 뜨게 된 것은 Supervised Learning (지도 학습: (input, output-answer) pair가 있는 데이터를 가지고 학습) 을 특히 잘 해냈기 때문
  - 종류: Standard NN (numerical input features), CNN(image), RNN(time series)
  - structured data, unstructured data
- 딥러닝이 taking off 한 이유
  > <img width="600" alt="Screen Shot 2021-08-18 at 8 59 18 PM" src="https://user-images.githubusercontent.com/6873655/129894350-c24612cd-8fb0-4908-835b-909cc4fbb762.png">
  - 또한, data, computation, algorithms의 혁신 -> idea,code, experiment 싸이클의 순환이 빨라짐

**Comment by KimDahye** [comment]
- [x] Neural networks & Deep Learning
  - [x] week 1 introduction to deep learning
     08.18(수) 발표자 - 김다혜
  - [x] week 2 neural networks basics
     08.25 (수) 발표자 - 신윤서
  - [x]  week 3 shallow neural networks
     09.01 (수) 발표자 - ~~Ellen~~ 고예슬
  - [x] week 4 deep neural networks
     09.08 (수) 발표자 - 김다혜
- [x] Improving Deep Neural Networks: Hyperparameter tuning, Regularization and Optimization
  - [x] week 1 practical aspects of deep learning
     09.15 (수) 발표자 - 서원국
  - [x] week 2 optimization algorithms 
     09.29 (수) 발표자 - Ellen // 추석연휴 쉬고 다시 시작
  - [x] week 3  hyperparameter tuning, Batch Normalization and Programming Frameworks
     10.06 (수) 발표자 - 신윤서
- [x] Structuring Machine Learning Projects
  - [x] week 1 - ML strategy 1
     10.13 (수) 발표자 - Ellen
  - [x] week 2 - ML strategy 2
     10.20 (수) 발표자 - Ellen
- [x] Convolutional Neural Networks
  - [x] week 1 Foundations of Convolutional Neural Networks
     10.27 (수) 발표자 - 서원국
  - [x] week 2 Deep convolutional models : case studies
     11.03 (수) 발표자 - ~~고예슬~~ Ellen
  - [x] week 3 Object detection
     11.10 (수) 발표자 - 김다혜
  - [x] week 4 Special applications : face recognition & neural style transfer
     11.17 (수) 발표자 - 최경호
- [ ] Sequence Models
  - [ ] week 1 - Recurrent Neural Networks
     11.24 (수) 발표자 - 고예슬
  - [ ] week 2 - Natural Language Processing & Word Embeddings
     12.01 (수) 발표자 - 최경호
  - [ ] week 3 - Sequence models & Attention mechanism
     12.08 (수) 발표자 - 최경호
  - [ ] week 4 - Transformer Network
     12.15 (수) 발표자 - 김다혜

**Comment by ellenseon-tech** [comment]
> * Neural networks & Deep Learning
>   
>   * week 1 introduction to deep learning
>     8.18(수) 발표자 - 김다혜
>   * week 2 neural networks basics
>     0.00 (수) 발표자 - 신윤서
>   * week 3 shallow neural networks
>     0.00 (수) 발표자 -
>   * week 4 deep neural networks
>     0.00 (수) 발표자 -
> * Improving Deep Neural Networks: Hyperparameter tuning, Regularization and Optimization
>   
>   * week 1 practical aspects of deep learning
>     0.00 (수) 발표자 -
>   * week 2 optimization algorithms
>     0.00 (수) 발표자 -
>   * week 3  hyperparameter tuning, Batch Normalization and Programming Frameworks
>     0.00 (수) 발표자 -
> * Structuring Machine Learning Projects
>   
>   * week 1 - ML strategy 1
>     0.00 (수) 발표자 - Ellen
>   * week 2 - ML strategy 2
>     0.00 (수) 발표자 - Ellen
> * Convolutional Neural Networks
>   
>   * week 1 Foundations of Convolutional Neural Networks
>     0.00 (수) 발표자 -
>   * week 2 Deep convolutional models : case studies
>     0.00 (수) 발표자 -
>   * week 3 Object detection
>     0.00 (수) 발표자 -
>   * week 4 Special applications : face recognition & neural style transfer
>     0.00 (수) 발표자 -
> * Sequence Models
>   
>   * week 1 - Recurrent Neural Networks
>     0.00 (수) 발표자 -
>   * week 2 - Natural Language Processing & Word Embeddings
>     0.00 (수) 발표자 -
>   * week 3 - Sequence models & Attention mechanism
>     0.00 (수) 발표자 -

**Comment by yeslkoh** [comment]
Neural networks & Deep Learning

week 1 introduction to deep learning
8.18(수) 발표자 - 김다혜
week 2 neural networks basics
0.00 (수) 발표자 - 신윤서
week 3 shallow neural networks
0.00 (수) 발표자 -
week 4 deep neural networks
0.00 (수) 발표자 -
Improving Deep Neural Networks: Hyperparameter tuning, Regularization and Optimization

week 1 practical aspects of deep learning
0.00 (수) 발표자 -
week 2 optimization algorithms
0.00 (수) 발표자 -
week 3 hyperparameter tuning, Batch Normalization and Programming Frameworks
0.00 (수) 발표자 -
Structuring Machine Learning Projects

week 1 - ML strategy 1
0.00 (수) 발표자 - Ellen
week 2 - ML strategy 2
0.00 (수) 발표자 - Ellen
Convolutional Neural Networks

week 1 Foundations of Convolutional Neural Networks
0.00 (수) 발표자 -
week 2 Deep convolutional models : case studies
0.00 (수) 발표자 - yskoh
week 3 Object detection
0.00 (수) 발표자 -
week 4 Special applications : face recognition & neural style transfer
0.00 (수) 발표자 -
Sequence Models

week 1 - Recurrent Neural Networks
0.00 (수) 발표자 - yskoh
week 2 - Natural Language Processing & Word Embeddings
0.00 (수) 발표자 - yskoh
week 3 - Sequence models & Attention mechanism
0.00 (수) 발표자 -

**Comment by jasmine-shin** [comment]
Neural networks & Deep Learning
week 1 introduction to deep learning
8.18(수) 발표자 - 김다혜
week 2 neural networks basics
8.25 (수) 발표자 - 신윤서
week 3 shallow neural networks
0.00 (수) 발표자 -
week 4 deep neural networks
9.08 (수) 발표자 - 신윤서
Improving Deep Neural Networks: Hyperparameter tuning, Regularization and Optimization
week 1 practical aspects of deep learning
9.15 (수) 발표자 -  신윤서
week 2 optimization algorithms
0.00 (수) 발표자 -
week 3 hyperparameter tuning, Batch Normalization and Programming Frameworks
0.00 (수) 발표자 -
Structuring Machine Learning Projects
week 1 - ML strategy 1
0.00 (수) 발표자 - Ellen
week 2 - ML strategy 2
0.00 (수) 발표자 - Ellen
Convolutional Neural Networks
week 1 Foundations of Convolutional Neural Networks
0.00 (수) 발표자 -
week 2 Deep convolutional models : case studies
0.00 (수) 발표자 - 고예슬
week 3 Object detection
0.00 (수) 발표자 -
week 4 Special applications : face recognition & neural style transfer
0.00 (수) 발표자 -
Sequence Models
week 1 - Recurrent Neural Networks
0.00 (수) 발표자 - 고예슬
week 2 - Natural Language Processing & Word Embeddings
0.00 (수) 발표자 - 고예슬
week 3 - Sequence models & Attention mechanism
0.00 (수) 발표자 -
week 4 -
0.00 (수) 발표자 -

**Comment by ellenseon-is-fantastic** [comment]
> * Neural networks & Deep Learning
>   
>   * week 1 introduction to deep learning
>     8.18(수) 발표자 - 김다혜
>   * week 2 neural networks basics
>     0.00 (수) 발표자 - 신윤서
>   * week 3 shallow neural networks
>     0.00 (수) 발표자 - Ellen
>   * week 4 deep neural networks
>     0.00 (수) 발표자 - Ellen
> * Improving Deep Neural Networks: Hyperparameter tuning, Regularization and Optimization
>   
>   * week 1 practical aspects of deep learning
>     0.00 (수) 발표자 -
>   * week 2 optimization algorithms
>     0.00 (수) 발표자 -
>   * week 3  hyperparameter tuning, Batch Normalization and Programming Frameworks
>     0.00 (수) 발표자 -
> * Structuring Machine Learning Projects
>   
>   * week 1 - ML strategy 1
>     0.00 (수) 발표자 - Ellen
>   * week 2 - ML strategy 2
>     0.00 (수) 발표자 - Ellen
> * Convolutional Neural Networks
>   
>   * week 1 Foundations of Convolutional Neural Networks
>     0.00 (수) 발표자 -
>   * week 2 Deep convolutional models : case studies
>     0.00 (수) 발표자 - 고예슬
>   * week 3 Object detection
>     0.00 (수) 발표자 -
>   * week 4 Special applications : face recognition & neural style transfer
>     0.00 (수) 발표자 -
> * Sequence Models
>   
>   * week 1 - Recurrent Neural Networks
>     0.00 (수) 발표자 - 고예슬
>   * week 2 - Natural Language Processing & Word Embeddings
>     0.00 (수) 발표자 - 고예슬
>   * week 3 - Sequence models & Attention mechanism
>     0.00 (수) 발표자 -
>   * week 4 -
>     0.00 (수) 발표자 -

**Comment by ellenseon-tech** [comment]
> Neural networks & Deep Learning
> week 1 introduction to deep learning
> 8.18(수) 발표자 - 김다혜
> week 2 neural networks basics
> 8.25 (수) 발표자 - 신윤서
> week 3 shallow neural networks
> 0.00 (수) 발표자 - Ellen
> week 4 deep neural networks
> 9.08 (수) 발표자 - 신윤서
> Improving Deep Neural Networks: Hyperparameter tuning, Regularization and Optimization
> week 1 practical aspects of deep learning
> 9.15 (수) 발표자 - 신윤서
> week 2 optimization algorithms
> 0.00 (수) 발표자 - Ellen
> week 3 hyperparameter tuning, Batch Normalization and Programming Frameworks
> 0.00 (수) 발표자 -
> Structuring Machine Learning Projects
> week 1 - ML strategy 1
> 0.00 (수) 발표자 - Ellen
> week 2 - ML strategy 2
> 0.00 (수) 발표자 - Ellen
> Convolutional Neural Networks
> week 1 Foundations of Convolutional Neural Networks
> 0.00 (수) 발표자 -
> week 2 Deep convolutional models : case studies
> 0.00 (수) 발표자 - 고예슬
> week 3 Object detection
> 0.00 (수) 발표자 -
> week 4 Special applications : face recognition & neural style transfer
> 0.00 (수) 발표자 -
> Sequence Models
> week 1 - Recurrent Neural Networks
> 0.00 (수) 발표자 - 고예슬
> week 2 - Natural Language Processing & Word Embeddings
> 0.00 (수) 발표자 - 고예슬
> week 3 - Sequence models & Attention mechanism
> 0.00 (수) 발표자 -
> week 4 -
> 0.00 (수) 발표자 -

**Comment by kukugi** [comment]
Neural networks & Deep Learning
week 1 introduction to deep learning
8.18(수) 발표자 - 김다혜
week 2 neural networks basics
8.25 (수) 발표자 - 신윤서
week 3 shallow neural networks
0.00 (수) 발표자 - Ellen
week 4 deep neural networks
9.08 (수) 발표자 - 신윤서
Improving Deep Neural Networks: Hyperparameter tuning, Regularization and Optimization
week 1 practical aspects of deep learning
9.15 (수) 발표자 - 신윤서
week 2 optimization algorithms
0.00 (수) 발표자 - Ellen
week 3 hyperparameter tuning, Batch Normalization and Programming Frameworks
0.00 (수) 발표자 - 서원국
Structuring Machine Learning Projects
week 1 - ML strategy 1
0.00 (수) 발표자 - Ellen
week 2 - ML strategy 2
0.00 (수) 발표자 - Ellen
Convolutional Neural Networks
week 1 Foundations of Convolutional Neural Networks
0.00 (수) 발표자 -
week 2 Deep convolutional models : case studies
0.00 (수) 발표자 - 고예슬
week 3 Object detection
0.00 (수) 발표자 - 서원국
week 4 Special applications : face recognition & neural style transfer
0.00 (수) 발표자 -
Sequence Models
week 1 - Recurrent Neural Networks
0.00 (수) 발표자 - 고예슬
week 2 - Natural Language Processing & Word Embeddings
0.00 (수) 발표자 - 고예슬
week 3 - Sequence models & Attention mechanism
0.00 (수) 발표자 - 서원국
week 4 -
0.00 (수) 발표자 -

**Comment by kingheadcat** [comment]
Neural networks & Deep Learning
 week 1 introduction to deep learning
08.18(수) 발표자 - 김다혜
 week 2 neural networks basics
08.25 (수) 발표자 - 신윤서
 week 3 shallow neural networks
09.01 (수) 발표자 - Ellen 고예슬
 week 4 deep neural networks
09.08 (수) 발표자 - 김다혜
 Improving Deep Neural Networks: Hyperparameter tuning, Regularization and Optimization
 week 1 practical aspects of deep learning
09.15 (수) 발표자 - 서원국
 week 2 optimization algorithms
09.29 (수) 발표자 - Ellen // 추석연휴 쉬고 다시 시작
 week 3 hyperparameter tuning, Batch Normalization and Programming Frameworks
10.06 (수) 발표자 - 신윤서
 Structuring Machine Learning Projects
 week 1 - ML strategy 1
10.13 (수) 발표자 - Ellen
 week 2 - ML strategy 2
10.20 (수) 발표자 - Ellen
 Convolutional Neural Networks
 week 1 Foundations of Convolutional Neural Networks
10.27 (수) 발표자 - 서원국
 week 2 Deep convolutional models : case studies
11.03 (수) 발표자 - 고예슬 Ellen
 week 3 Object detection
11.10 (수) 발표자 - 김다혜
 week 4 Special applications : face recognition & neural style transfer
11.17 (수) 발표자 - 최경호
 Sequence Models
 week 1 - Recurrent Neural Networks
11.24 (수) 발표자 - 고예슬
 week 2 - Natural Language Processing & Word Embeddings
12.01 (수) 발표자 - 최경호
 week 3 - Sequence models & Attention mechanism
12.08 (수) 발표자 - 최경호
 week 4 - Transformer Network
12.15 (수) 발표자 - 김다혜
